test/functional/specs/Identity/demdexFallback.js

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { t } from "testcafe";
+import createNetworkLogger from "../../helpers/networkLogger/index.js";
+import createFixture from "../../helpers/createFixture/index.js";
+import {
+  compose,
+  orgMainConfigMain,
+  thirdPartyCookiesEnabled,
+} from "../../helpers/constants/configParts/index.js";
+import createAlloyProxy from "../../helpers/createAlloyProxy.js";
+
+const networkLogger = createNetworkLogger();
+const config = compose(orgMainConfigMain, thirdPartyCookiesEnabled);
+
+createFixture({
+  title: "Demdex Fallback Behavior",
+  requestHooks: [networkLogger.edgeEndpointLogs],
+});
+
+test("Continues collecting data when demdex is blocked", async () => {
+  const alloy = createAlloyProxy();
+
+  // Block demdex.net
+  await t.eval(() => {
```

Comment:
I think it is better if this test uses a [Testcafe Request Mock](https://testcafe.io/documentation/402844/guides/advanced-guides/detect-the-client-browser-and-platform?search#handle-ads-on-mobile-devices), instead of modifying the on-page `fetch` in the same way the unit test does